lib/puppet/provider/keycloak_client_protocol_mapper/kcadm.rb

```diff
@@ -105,7 +105,7 @@ def create
     if resource[:type] == 'oidc-usermodel-property-mapper' || resource[:type] == 'saml-user-property-mapper'
       data[:config][:'user.attribute'] = resource[:user_attribute] if resource[:user_attribute]
     end
-    if ['oidc-usermodel-property-mapper', 'oidc-group-membership-mapper'].include?(resource[:type])
+    if ['oidc-usermodel-client-role-mapper', 'oidc-group-membership-mapper'].include?(resource[:type])
```

Comment:
I think this is causing tests to fail because you replaced property-mapper with client-role mapper rather than adding client-role mapper:

```suggestion
    if ['oidc-usermodel-client-role-mapper', 'oidc-usermodel-property-mapper', 'oidc-group-membership-mapper'].include?(resource[:type])
```

Reply:
Uh, of course. One bug per line of code :smile:.